docs/dashboards/dashboard-optimization.md

```diff
@@ -0,0 +1,53 @@
+---
+id: dashboard-optimization
+title: Dashboard Optimization
+sidebar_label: Dashboard Optimization
+description: Learn more about the dashboard optimization to improve the efficiency of Dashboard searches.
+---
+
+Optimization improves the efficiency of Dashboard searches. 
+
+Sumo automatically optimizes a dashboard panel if it loads continuously for five minutes. If you pause a query before five minutes has elapsed, and then resume it, the timer restarts.
+
+When a query has loaded for five minutes without completing, Sumo creates a scheduled view for the query, and modifies the query to use that view. Queries that were created with a scheduled view are considered to be already optimized, so Sumo does not attempt additional optimization.
+
+## Optimization icons
+
+The icon in the upper right corner of each Dashboard Panel shows the current optimization status of that Panel.
```

Comment:
Unless you're referring to a UI element (like "Click **Panel**"), ok to leave this lowercase.

```suggestion
The icon in the upper right corner of each dashboard panel shows the current optimization status of that panel.
```